Nourfabouz/innnn
# Bug fix/mariadb

# MariaDB Configuration Update

This pull request introduces necessary changes to the Dockerfile and startup script for our MariaDB service. The changes ensure the proper setup and configuration of MariaDB as per our application's requirements.

## Dockerfile

The Dockerfile has been updated to correctly copy the startup script into the Docker image during the build process. The script is also made executable within the Dockerfile itself.

## Startup Script

The startup script has been updated with the following:

- Starts the MariaDB service.
- Sets the correct authentication methods.
- Creates the necessary database (if it doesn't already exist).
- Checks for the existence of the desired user, creating it if necessary.
- Grants appropriate privileges to the user on the database.
- Changes the root user's password.
- Flushes the privileges to ensure the changes take effect.

The changes in the script help in setting up the MariaDB server to meet the specific needs of our application.

## Testing

To test the changes introduced in this pull request:

1. Pull this branch.
2. Build the project with `make build` command.
3. Verify that the WordPress website is running properly.
